ci-operator/config/openshift/verification-tests/openshift-verification-tests-master__installation-nightly-4.14.yaml

```diff
@@ -150,6 +150,26 @@ tests:
     test:
     - chain: cucushift-installer-check-cluster-health
     workflow: cucushift-installer-rehearse-azure-ipi-publish-mixed-apiserver-ingress-external
+- as: azure-ipi-oidc-managed-identity-system-f28
```

Comment:
In a common azure install, the azure credentials would be saved into kube-system, I do not sure what does it look like when using managed-identify, I am not sure if that cause some unexpected behaviors for e2e testing, to be safe, I suggest to move this to common openshift ci.

Reply:
managed-identity here only impacts installer, and installer requires cco in manual mode if using managed-identity, so I think it should no impact for e2e testing.

Reply:
okay.